### PR TITLE
Install wget in some tests using it

### PR DESCRIPTION
**Pull request by Vogtinator**
Those tests are run in JeOS as well, which does not have wget installed.

Validation run: http://10.160.67.86/tests/191

### CODE REVIEW

File: tests/console/rabbitmq.pm [hunk 1]
```diff
@@ -20,7 +20,7 @@ use utils 'systemctl';
 
 sub run {
     select_console 'root-console';
-    assert_script_run('zypper -n in rabbitmq-server');
+    assert_script_run('zypper -n in rabbitmq-server wget');
```
Comment (okurz) [inline tests/console/rabbitmq.pm]: I recommend to move the installation of wget into the next zypper call to install "python-pika" which we install to execute the test, this line here can be considered part of the test assertions if you get my meaning

File: tests/console/zbar.pm [hunk 2]
```diff
@@ -19,7 +19,7 @@ use utils;
 
 sub run {
     select_console 'user-console';
-    assert_script_sudo("zypper -n in zbar");
+    assert_script_sudo("zypper -n in wget zbar");
```
Comment (okurz) [inline tests/console/zbar.pm]: Also here, might be more explicit to install "wget" in its own call to make the test dependency more clear

Comment (Vogtinator) [reply tests/console/zbar.pm]: Not really - the assert not only means "verify that the product works" but also "verify that the test can continue".

I don't see any value in separating it.

File: tests/console/rabbitmq.pm (hunk 1 above)
Comment (Vogtinator) [reply tests/console/rabbitmq.pm]: Done.

**Pull request merged**